integration-cli/docker_cli_prune_unix_test.go

```diff
@@ -90,3 +91,85 @@ func (s *DockerDaemonSuite) TestPruneImageDangling(c *check.C) {
 	c.Assert(err, checker.IsNil)
 	c.Assert(strings.TrimSpace(out), checker.Not(checker.Contains), id)
 }
+
+func (s *DockerSuite) TestPruneContainerUntil(c *check.C) {
+	out, _ := dockerCmd(c, "run", "-d", "busybox")
+	id1 := strings.TrimSpace(out)
+	c.Assert(waitExited(id1, 5*time.Second), checker.IsNil)
+
+	until := daemonUnixTime(c)
+
+	out, _ = dockerCmd(c, "run", "-d", "busybox")
+	id2 := strings.TrimSpace(out)
+	c.Assert(waitExited(id2, 5*time.Second), checker.IsNil)
+
+	out, _ = dockerCmd(c, "container", "prune", "--force", "--filter", "until="+until)
+	c.Assert(strings.TrimSpace(out), checker.Contains, id1)
+	c.Assert(strings.TrimSpace(out), checker.Not(checker.Contains), id2)
+
+	out, _ = dockerCmd(c, "ps", "-a", "-q", "--no-trunc")
+	c.Assert(strings.TrimSpace(out), checker.Not(checker.Contains), id1)
+	c.Assert(strings.TrimSpace(out), checker.Contains, id2)
+}
+
+func (s *DockerDaemonSuite) TestPruneImageUntil(c *check.C) {
+	c.Assert(s.d.StartWithBusybox(), checker.IsNil)
```

Comment:
If you rebase, you should be able to do `s.d.StartWithBusybox(c)` without the `Assert`.